views/partials/what_to_do_svg.hbs

```diff
@@ -1,6 +1,6 @@
-<svg role="img" width="42px" height="42px" viewBox="0 0 42 42" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg role="img" width="42px" height="42px" viewBox="0 0 42 42">
   <defs>
-    <linearGradient x1="100%" y1="0%" x2="0%" y2="100%" id="linearGradient-1">
+    <linearGradient x1="100%" y1="0%" x2="0%" y2="100%" id="linearGradient-{{ number }}">
```

Comment:
I feel like you need to need to tweak L11 below (sorry, it seems to fall outside the scope of this PR and I can't comment on that line directly):

```svg
<circle stroke="url(#linearGradient-1)" stroke-width="2" cx="20" cy="20" r="20"></circle>
```